src/Microsoft.DotNet.Watcher.Tools/Microsoft.DotNet.Watcher.Tools.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <AssemblyName>dotnet-watch</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <Description>Command line tool to watch for source file changes during development and restart the dotnet command.</Description>
+    <PackageTags>dotnet;watch</PackageTags>
+    <PackageType>DotnetCliTool</PackageType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Shared\**\*.cs" />
+    <Compile Include="**\*.cs" />
+    <EmbeddedResource Include="**\*.resx" />
+    <EmbeddedResource Include="dotnetwatch.targets" LogicalName="dotnetwatch.targets" />
+    <Content Include="tools\*.targets" CopyToOutputDirectory="PreserveNewest" Pack="true" PackagePath="%(Identity)" />
+    <None Include="$(ProjectRuntimeConfigFilePath)" Pack="true" PackagePath="lib/$(TargetFramework)/" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.0.0" />
+    <PackageReference Include="Microsoft.NET.Sdk" Version="1.0.0-*" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.NETCore.App" Version="1.0.0" />
+
+    <!-- Restore this when https://github.com/dotnet/sdk/issues/115 is resolved -->
+    <!--<PackageReference Include="Microsoft.Extensions.Process.Sources" Version="1.1.0-*" PrivateAssets="All" />-->
+
+    <!-- ExcludeAssets="build" to workaround https://github.com/dotnet/corefx/issues/12913 -->
+    <PackageReference Include="Microsoft.DiaSymReader.Native" Version="1.4.0" ExcludeAssets="build" PrivateAssets="All" />
```

Comment:
It was a workaround for https://github.com/dotnet/roslyn/issues/15137. Just checked again and the workaround can be removed now.